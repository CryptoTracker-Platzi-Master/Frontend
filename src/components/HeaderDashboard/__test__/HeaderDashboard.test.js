// import React from "react";
// import { BrowserRouter } from "react-router-dom";

// import "@testing-library/jest-dom/extend-expect";
// import { render } from "@testing-library/react";
// import { HeaderDashboard } from "../";

// describe("[ component / HeaderDashboard ]", () => {
//   describe("when  `HeaderDashboard` is mounted", () => {
//     it("[`HeaderDashboard`]", () => {
//       const { asFragment } = render(
//         <BrowserRouter>
//           <HeaderDashboard />
//         </BrowserRouter>
//       );
//       expect(asFragment()).toMatchSnapshot();
//     });
//   });
// });
