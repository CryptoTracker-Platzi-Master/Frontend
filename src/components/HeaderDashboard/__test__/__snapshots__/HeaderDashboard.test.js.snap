// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[ component / HeaderDashboard ] when  \`HeaderDashboard\` is mounted [\`HeaderDashboard\`] 1`] = `
<DocumentFragment>
  <header
    class="header-dashboard"
  >
    <figure
      class="header-dashboard__container-img"
    >
      <img
        alt="logo"
        class="header-dashboard__container-img--img"
        src="img-perfile.png"
      />
    </figure>
    <p
      class="header-dashboard--name-user"
    >
      Hi, 
      <span>
        Leonardo
      </span>
    </p>
    <nav
      class="header-dashboard__nav"
    >
      <a
        href="/"
      >
        <i
          class="fas fa-sign-out-alt header-dashboard__nav--logout"
        />
      </a>
    </nav>
  </header>
</DocumentFragment>
`;
